src/Microsoft.ML.AutoML/Utils/UserInputValidationUtil.cs

```diff
@@ -20,6 +20,8 @@ internal static class UserInputValidationUtil
         private const string TextColumnPurposeName = "text";
         private const string IgnoredColumnPurposeName = "ignored";
         private const string SamplingKeyColumnPurposeName = "sampling key";
+        private const string MatrixColumnIndexColumnName = "matrix column index";
```

Comment:
I thought we were going to rename these to "User ID" and "Item ID"?

Reply:
I see, MatrixFactorization calls these columns those names.  Feel free to ignore this.

---
In reply to: [335649246](https://github.com/dotnet/machinelearning/pull/4246#discussion_r335649246) [](ancestors = 335649246)

Reply:
Please refer to https://github.com/dotnet/machinelearning/pull/4246#discussion_r335316079

I want to make sure nobody is against the UserId and ItemId naming before I finalize rename.
@justinormont would you be OK with using UserId/ItemId?